### PR TITLE
Integrate new UI

### PR DESCRIPTION
**Pull request by uranusjr**
需要大家的幫助把所有頁面補齊。Menu 上面有的頁面就是需要加，當測試過了就是完成了（應該）。

### CODE REVIEW

File: src/static/pycontw-2018/scripts/app.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import './polyfills'
+
+import {Application} from 'stimulus'
```
Comment (keitheis) [inline src/static/pycontw-2018/scripts/app.js]: 騷包 🧞‍♂️

File: src/core/context_processors.py [hunk 2]
```diff
@@ -1,15 +1,25 @@
 from django.conf import settings
+from django.utils.translation import get_language
 
 from sponsors.models import Sponsor
 
 
+def _build_google_form_url(uid):
+    return 'https://docs.google.com/forms/d/e/{uid}/viewform'.format(uid=uid)
```
Comment (keitheis) [inline src/core/context_processors.py]: Within Python 3.6, alternative:
```python
+    return f'https://docs.google.com/forms/d/e/{uid}/viewform'
```

File: src/core/context_processors.py [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
         'GA_TRACK_ID': settings.GA_TRACK_ID,
         'KKTIX_URL': 'https://pycontw.kktix.cc/events/pycontw2017',
-        'VOLUNTEER_FORM_URL': (
-            'https://docs.google.com/forms/d/e/'
-            '1FAIpQLSfnTEpD_I74Yeji6GqfI-YSZoYgg7Ax-YCEG5PNHrBusTgn4Q/viewform'
+        'SPONSOR_FORM_URL': _build_google_form_url(sponsor_id),
+        'VOLUNTEER_FORM_URL': _build_google_form_url(
+            '1FAIpQLSfnTEpD_I74Yeji6GqfI-YSZoYgg7Ax-YCEG5PNHrBusTgn4Q',
         ),
     }
```
Comment (keitheis) [inline src/core/context_processors.py]: Updated version of urls of sponsorship form:
```
2018 贊助商申請表單（中文）
https://goo.gl/P3zqmL

2018 贊助商申請表單（英文）
https://goo.gl/MLZTWU
```

`def _build_google_form_url` can be gone then. Not sure they worth a def.

File: src/templates/pycontw-2018/base.html [hunk 4]
```diff
@@ -12,22 +12,23 @@
...
 {% endblock titletag %}
 
-<link rel="icon" type="image/png" href="{% static 'pycontw-2018/images/favicon.png' %}">
+<link rel="shortcut icon" href="{% static 'pycontw-2018/assets/favicon.ico' %}">
+<link rel="icon" type="image/png" href="{% static 'pycontw-2018/assets/favicon-16x16.png' %}" sizes="16x16">
+<link rel="icon" type="image/png" href="{% static 'pycontw-2018/assets/favicon-32x32.png' %}" sizes="32x32">
 
 {% block meta_og %}
 <meta property="og:description" content="{% block meta_description %}{% endblock meta_description %}">
 <meta property="og:image" content="{{ request.scheme }}://{{ request.get_host }}{% static 'pycontw-2018/images/og-image.png' %}">
```
Comment (keitheis) [inline src/templates/pycontw-2018/base.html]: Just noting: image file `pycontw-2018/images/og-image.png` is deleted in this PR.

File: src/static/pycontw-2018/scripts/app.js (hunk 1 above)
Comment (uranusjr) [reply src/static/pycontw-2018/scripts/app.js]: 我發現它其實很好用欸，感覺像 jQuery for 2018。

**Comment by uranusjr** [comment]
還需要搞定 #415 的文字問題，不過是可以先進 master 等翻譯了。

File: src/events/tests/conftest.py [hunk 5]
```diff
@@ -1,18 +1,37 @@
...
+from django.utils.timezone import is_naive
 
 from events.models import Location, SponsoredEvent, Time
 
 
+cst = pytz.timezone('Asia/Taipei')
+
+
+@pytest.fixture
+def get_time(day):
```
Comment (keitheis) [inline src/events/tests/conftest.py]: Have hint in function name or function doc or comment on why it's needed or what is its purpose?

File: src/templates/pycontw-2018/base.html (hunk 4 above)
Comment (uranusjr) [reply src/templates/pycontw-2018/base.html]: Need to be re-added (Carol provided the source on Slack).

File: src/core/context_processors.py (hunk 3 above)
Comment (uranusjr) [reply src/core/context_processors.py]: There’s still the volunteer form. But I can create a short URL for it too.

**Comment by uranusjr** [comment]
都可以，到了 master 我還是可以繼續改上面的東西。

**Comment by uranusjr** [comment]
Going into master!

**Pull request closed without merging**